.travis.yml

```diff
@@ -12,3 +12,28 @@ script:
   - node dist/node/loader.entry.js
 after_success:
   - npm run report-coverage
+branches:
+  only:
+    - master
+    - /^v[0-9]/
```

Comment:
Pretty sure this is actually restricting all builds/tests to only those branches, not just deploys, right?